app/controllers/state_file/questions/income_review_controller.rb

```diff
@@ -1,6 +1,7 @@
 module StateFile
   module Questions
     class IncomeReviewController < QuestionsController
+      include ReturnToReviewConcern
```

Comment:
looks good to me -- can we add the spec to test `it_behaves_like :return_to_review_concern`?